src/parser/hevc_parser.cpp

```diff
@@ -84,11 +84,11 @@ rocDecStatus HevcVideoParser::ParseVideoData(RocdecSourceDataPacket *p_data) {
         }
 
         // Init Roc decoder for the first time or reconfigure the existing decoder
-        if (new_sps_activated_) {
+        if (new_seq_activated_) {
```

Comment:
why this is changed from sps to seq?

Reply:
Because this new_seq_activated is now applied to AV1/AVC/HEVC. seq means new sequence header for AV1 and new SPS for AVC/HEVC.